appveyor.yml

```diff
@@ -161,6 +162,7 @@ for:
           Exit-AppVeyorBuild
           return
       }
+      Push-AppveyorArtifact .\GitExtensionsVSIX\bin\Release\GitExtensionsVSIX.vsix
```

Comment:
❓ Why is this line needed? It seems like a duplicate of the line above:

https://github.com/gitextensions/gitextensions/blob/da096174401e285472afd077418e9f9d584d58cb/appveyor.yml#L121

Reply:
We have 2 distinct paths in the appveyor:
* all non-release branches: https://github.com/gitextensions/gitextensions/blob/master/appveyor.yml#L83-L90
* release branches: https://github.com/gitextensions/gitextensions/blob/master/appveyor.yml#L122-L126

And we publish different artifacts for each.
In fact we publish different artifacts for release braches depending on whether they are from an official release (i.e. signed) or not.

Reply:
It may be possible to add an "artifacts: path:" section for release too, I choose to follow the pattern for release. But it is hard to test.